CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Fix warning about using absolute paths when overwriting an fbx file.
 - Fix exporting zero scale not importing correctly in 3ds Max.
 - Fix window size cutting off text in export window.
+- Fix export to new folder on Mac.
```

Comment:
I think we should specify that it was not possible to create a new folder when choosing the prefab/export paths.